crates/nu-cli/src/commands/autoview.rs

```diff
@@ -77,6 +77,13 @@ pub fn autoview(context: RunnableContext) -> Result<OutputStream, ShellError> {
     let binary = context.get_command("binaryview");
     let text = context.get_command("textview");
     let table = context.get_command("table");
+    let no_auto_pivot = config::config(Tag::unknown())?
+        .get("no_auto_pivot")
+        .map(|s| match s.value.expect_string() {
+            "true" => true,
+            _ => false,
+        })
```

Comment:
We have builtin boolean types, so perhaps we should ensure a boolean goes into this config value instead of a string? @jonathandturner may have some opinions here.

Reply:
This also makes me think we should define a schema for our config somewhere! We should do so with code so that we can produce docs/help automatically 🙂

Reply:
I agree that it should be a boolean, I reused code from cli.rs where ctrlc_exit is retrieved from the config, so I didn't catch that i could use a boolean. 

I've tried something like
```
let no_auto_pivot = match config::config(Tag::unknown())?.get("no_auto_pivot") {
    Some(val) => val.is_true(),
    _ => false,
};
```
and
```
let no_auto_pivot = match config::config(Tag::unknown())?.get("no_auto_pivot") {
    Some(val) => val.as_bool(),
    _ => Ok(false),
}.unwrap_or(false);
```
but it seems that always gets back false no matter what I set the config value to. I've tried 'true' 'false' '0' '1' and an empty value. Maybe config stores everything as a Primitive::String?

Reply:
`$true` and `$false` are the special boolean values in nushell. I guess we haven't really documented those? 😅

Reply:
if you have any problems with supporting the builtin booleans, I'd say reuse the pattern from ctrlc_exit like you've done, and we'll land that.

Reply:
I got the built in booleans working, thanks for the help!
`config --set [no_auto_pivot $true]`